src/ApiPort/Resources/LocalizedStrings.resx

```diff
@@ -248,6 +248,11 @@ gives a summary of issues and possible steps forward.
   -r, --resultFormat=VALUE   The report output format
   -p, --showNonPortableApis  Calculate non-portable APIs
   -b, --showBreakingChanges  Calculate breaking changes on full .NET Framework
+  -u, --onlyShowRuntimeIssues
+                             Only show the issues that would need to be 
+                             addressed if keeping existing target and 
+                             running on a newer framework version. This 
+                             will automatically show breaking changes.
```

Comment:
This help statement says that --u will imply --b, but it doesn't look like that's the case. You should either update argument parsing to make sure that happens, or update the definition of the OnlyShowRuntimeChanges flag to include the ShowBreakingChangesFlag.